src/tirith/logging.py

```diff
@@ -1,3 +1,13 @@
+"""
+Sets up logging configuration for the application.
+
+It provides a default logging configuration with different handlers and formatters
+to control the verbosity and formatting of log messages. The ``setup_logging`` function
+allows switching between verbose and normal logging modes.
+
+"""
```

Comment:
We can remove the `It provides a default logging configuration with different handlers and formatters
to control the verbosity and formatting of log messages. The ``setup_logging`` function
allows switching between verbose and normal logging modes.` part